diff.go

```diff
@@ -2,10 +2,13 @@ package reviewdog
 
 import (
 	"context"
+	"fmt"
 	"os/exec"
 	"sync"
 )
 
+var Y = fmt.Sprintf("test change %s")
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var Y should have comment or be unexported